### PR TITLE
Calculator assignemt

### PR DESCRIPTION
**Pull request by Vetchadines**

### CODE REVIEW

File: Calculator.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        switch (operators){
+
+            //addition operation
+            case '+':
+                output = num1 + num2;
+                System.out.println(num1 + "+" + num2 + "=" + output);
+                break;
+
+            // Subtraction Operation
+            case '_':
```
Comment (arunbhargav03) [inline Calculator.java]: change this to "-"

File: Calculator.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    public static void main(String[] args ) {
+
+        char operators;
+        Double num1, num2, output;
+
+        //creation of an object for scanner class
+        Scanner input = new Scanner(System.in);
+
+        // User entering the operator
+        System.out.println("Enter the operator: ");
```
Comment (arunbhargav03) [inline Calculator.java]: add what are the allowed operators

**Review by arunbhargav03 — CHANGES_REQUESTED** [review]

File: Calculator.java (hunk 1 above)
Comment (Vetchadines) [reply Calculator.java]: Done

**Review by arunbhargav03 — APPROVED** [review]

File: Calculator.java (hunk 2 above)
Comment (Vetchadines) [reply Calculator.java]: Done

**Pull request merged**